### PR TITLE
AFIT-101: new extended trg word(min bias), update for FT0 time offset collibration

### PR DESCRIPTION
**Pull request by afurs**
- Added extended FIT trg functionality + min bias for FT0
- Removed deprecated old calibration tasks and macro
- Additionally prepared common data formats for FIT digits (for next updates)

### CODE REVIEW

**Comment by afurs** [comment]
macos CICD issues unrelated

File: DataFormats/Detectors/FIT/FT0/src/DataFormatsFT0LinkDef.h [hunk 1]
```diff
@@ -55,4 +55,11 @@
 #pragma link C++ class o2::ft0::GlobalOffsetsInfoObject + ;
 #pragma link C++ class std::pair < std::vector < double>, std::vector < double>> + ;
 #pragma link C++ class o2::ft0::SlewingCoef + ;
+/*
+#include "DetectorsCommonDataFormats/DetID.h"
+#include "DataFormatsFIT/ChannelData.h"
+#pragma link C++ class o2::fit::ChannelData < o2::detectors::DetID::FT0> + ;
+#include "DataFormatsFIT/Digit.h"
+#pragma link C++ class o2::fit::Digit < o2::detectors::DetID::FT0> + ;
+*/
```
Comment (sawenzel) [inline DataFormats/Detectors/FIT/FT0/src/DataFormatsFT0LinkDef.h]: this is just a comment ... should probably not be committed

File: DataFormats/Detectors/FIT/FDD/src/DataFormatsFDDLinkDef.h [hunk 2]
```diff
@@ -36,5 +36,11 @@
 #pragma link C++ class o2::fdd::CTFHeader + ;
 #pragma link C++ class o2::fdd::CTF + ;
 #pragma link C++ class o2::ctf::EncodedBlocks < o2::fdd::CTFHeader, 8, uint32_t> + ;
-
+/*
+#include "DetectorsCommonDataFormats/DetID.h"
+#include "DataFormatsFIT/ChannelData.h"
+#pragma link C++ class o2::fit::ChannelData < o2::detectors::DetID::FDD> + ;
+#include "DataFormatsFIT/Digit.h"
+#pragma link C++ class o2::fit::Digit < o2::detectors::DetID::FDD> + ;
```
Comment (sawenzel) [inline DataFormats/Detectors/FIT/FDD/src/DataFormatsFDDLinkDef.h]: this is just a comment ... should probably not be committed

Comment (afurs) [reply DataFormats/Detectors/FIT/FDD/src/DataFormatsFDDLinkDef.h]: @sawenzel Hi, I turned off(commented) these in latest commits (for some reason I couldn't reproduce CICD issues on localhost). CICD takes too much time, so any cosmetics or minor hotfixes will go in next PR.

**Comment by afurs** [comment]
> Error while checking build/O2/fullCI for [d27f489](https://github.com/AliceO2Group/AliceO2/commit/d27f48919132204e54f09d67e8f8c0c3549ed247) at 2024-09-16 11:34:
> 
> ```
> ## sw/BUILD/O2-latest/log
> c++: error: unrecognized command-line option '--rtlib=compiler-rt'
> c++: error: unrecognized command-line option '--rtlib=compiler-rt'
> 
> 
> ## sw/BUILD/o2checkcode-latest/log
> --
> ========== List of errors found ==========
> ++ GRERR=0
> ++ grep -v clang-diagnostic-error error-log.txt
> ++ grep ' error:'
> /sw/SOURCES/O2/13514-slc8_x86-64/0/Detectors/GlobalTracking/src/MatchTOF.cxx:1150:16: error: 'signed char' to 'int' conversion; consider casting to 'unsigned char' first. [bugprone-signed-char-misuse]
> /sw/SOURCES/O2/13514-slc8_x86-64/0/Detectors/GlobalTrackingWorkflow/src/TOFMatchChecker.cxx:108:16: error: 'signed char' to 'int' conversion; consider casting to 'unsigned char' first. [bugprone-signed-char-misuse]
> /sw/SOURCES/O2/13514-slc8_x86-64/0/EventVisualisation/Workflow/src/EveWorkflowHelper.cxx:898:16: error: 'signed char' to 'int' conversion; consider casting to 'unsigned char' first. [bugprone-signed-char-misuse]
> /sw/SOURCES/O2/13514-slc8_x86-64/0/EventVisualisation/Workflow/src/EveWorkflowHelper.cxx:898:16: error: 'signed char' to 'int' conversion; consider casting to 'unsigned char' first. [bugprone-signed-char-misuse]
> ++ [[ 0 == 0 ]]
> ++ exit 1
> --
> ```
> 
> Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/13514/d27f48919132204e54f09d67e8f8c0c3549ed247/build_O2_fullCI/pretty.html).

Unrelated issue

**Pull request merged**